izanami-frontend/src/pages/projectSettings.tsx

```diff
@@ -77,9 +81,15 @@ export function ProjectSettings(props: { project: string; tenant: string }) {
                 .then(() => setInviting(false))
             }
             cancel={() => setInviting(false)}
+            invitedUsers={usersData?.map((user) => user.username) || []}
           />
         )}
-        <ProjectUsers tenant={tenant} project={project} />
+        <ProjectUsers
+          tenant={tenant}
+          project={project}
+          isLoading={isLoading}
```

Comment:
Since ProjectUsers is not handling the fetch part anymore, it should not be responsible for displaying loading indicator.

You should display loading indicator directly in this component.